tsconfig.json

```diff
@@ -1,12 +1,20 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es3",
```

Comment:
Why ES3 (I'm sure there's a good reason, just curious)?

Reply:
No good reason, just copied pusher-js. Not sure what makes most sense these days?

Reply:
I'd just leave it at ES5 unless there's a specific reason to use ES3. I don't think there are any browsers that we need to support that don't support ES5. Using ES3 probably also means that the output code is bigger and using utility functions to implement things that the browser already supports. Not a big deal though.